HW8_1.ipynb

```diff
@@ -0,0 +1,1611 @@
+{
```

Comment:
<p>I think the problem here is <code>%pycodestyle_o</code>  why the "_o"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_Calculus/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>